js/tests/MegaMenu/tests.js

```diff
@@ -71,4 +71,14 @@ QUnit.test("No links should have click handlers before the plugin is applied", f
         assert.ok(typeof this.onclick !== 'function', "No links in the mega menu have click handlers");
     });
 
-});
\ No newline at end of file
+});
+
+QUnit.test("CSS displays property of block on click.", function( assert ) {
```

Comment:
A couple of things here: 

- [ ] This test is within the module relating to WebTrends click handlers
- [ ] The description doesn't fully describe what is being tested